Adding examples for wavelength and detector. (#127)
sunpy/sunpy-soar: docs/how_to/wavelength.rst

.. _sunpy-soar-how-to-query-wavelength:

*************************************
How to query using Wavelength in SOAR
*************************************

``sunpy-soar`` provides a convenient methods to query Wavelength for all different instruments available in SOAR.
In this guide, we will demonstrate how we can query for Wavelength for different instruments.

> For instruments EUI, METIS and SOLOHI we get query results in form of wavelength,
> for instrument PHI we get query results in form of wavemin and wavemax and
> for instruments SPICE and STIX we do not get any meaningful data for wavelength.

Reviewer: ```suggestion
For instruments EUI, METIS and SOLOHI we get query results in form of wavelength, for instrument PHI we get query results in form of wavemin and wavemax and for instruments SPICE and STIX we do not get any meaningful data for wavelength.
```
We go for one sentence per line.

For instruments EUI, METIS and SOLOHI passing a single Wavelength
=================================================================

When a single wavelength is provided it is interpreted as the wavelength.

.. code-block:: python

> 
> 

Reviewer: ```suggestion

```

.. _sunpy-soar-how-to-query-wavelength:

*************************************
How to query using Wavelength in SOAR
*************************************

``sunpy-soar`` provides a convenient methods to query Wavelength for all different instruments available in SOAR.
In this guide, we will demonstrate how we can query for Wavelength for different instruments.
For instruments EUI, METIS and SOLOHI we get query results in form of wavelength, for instrument PHI we get query results in form of wavemin and wavemax and for instruments SPICE and STIX we do not get any meaningful data for wavelength.

For instruments EUI, METIS and SOLOHI passing a single Wavelength
=================================================================

When a single wavelength is provided it is interpreted as the wavelength.

> .. code-block:: python
>     >>> import astropy.units as u

Reviewer: ```suggestion
.. code-block:: python

    >>> import astropy.units as u
```

Not sure if this will fix the error

Author: I think it will.

         METIS metis-uv-image    L2 2023-02-01 01:30:48.680 2023-02-01 01:41:45.540     0.85      none      UVD      121.6
         METIS metis-uv-image    L2 2023-02-01 02:00:48.671 2023-02-01 02:11:44.213    12.64      none      UVD      121.6
         METIS metis-uv-image    L2 2023-02-01 02:30:48.661 2023-02-01 02:41:42.887    12.64      none      UVD      121.6
         METIS metis-uv-image    L2 2023-02-01 03:00:48.652 2023-02-01 03:11:41.560    12.64      none      UVD      121.6
         METIS metis-uv-image    L2 2023-02-01 03:30:48.643 2023-02-01 03:41:40.233    12.64      none      UVD      121.6
         METIS metis-uv-image    L2 2023-02-01 04:00:48.633 2023-02-01 04:11:38.907    12.64      none      UVD      121.6
         METIS metis-uv-image    L2 2023-02-01 04:30:38.163 2023-02-01 04:40:37.625    12.64      none      UVD      121.6
    <BLANKLINE>
    <BLANKLINE>

For instruments EUI, METIS and SOLOHI passing a range of Wavelength
===================================================================

When a range of wavelength is provided, it is interpreted as the wavemin and wavemax.

> .. code-block:: python
>     >>> wavelength = a.Wavelength(580 * u.AA, 640 * u.AA)

Reviewer: ```suggestion
.. code-block:: python

    >>> wavelength = a.Wavelength(580 * u.AA, 640 * u.AA)
```

.. _sunpy-soar-how-to-query-wavelength:

*************************************
How to query for Solar Orbiter data using the `Wavelength` attribute using `sunpy-soar`.
*************************************

> ``sunpy-soar`` provides a convenient methods to construct queries using `a.Wavelength` for several different remote sensing instruments available through the SOAR.

Reviewer: ```suggestion
``sunpy-soar`` provides a convenient methods to construct queries using ``a.Wavelength`` for several different remote sensing instruments available through the SOAR.
```

> In this guide, we will demonstrate how we can query data using `a.Wavelength` for different instruments.

Reviewer: ```suggestion
In this guide, we will demonstrate how we can query data using ``a.Wavelength`` for different instruments.
```

> For instruments EUI, METIS and SOLOHI we get query results in form of wavelength, for instrument PHI we get query results in form of wavemin and wavemax and for instruments SPICE and STIX we do not get any meaningful data for wavelength.

Reviewer 2: ```suggestion
For instruments EUI, METIS and SOLOHI we get query results in form of wavelength. However, at this time we cannot search for wavelengths for instruments SPICE, PHI, and STIX as this information is not yet available in the SOAR.
```

> For instruments EUI, METIS and SOLOHI we get query results in form of wavelength. However, at this time we cannot search for wavelengths for instruments SPICE, PHI, and STIX as this information is not yet available in the SOAR.

Reviewer: ```suggestion
For instruments EUI, METIS and SOLOHI we get query results in form of wavelength. 
However, at this time we cannot search for wavelengths for instruments SPICE, PHI, and STIX as this information is not yet available in the SOAR.
```

.. _sunpy-soar-how-to-query-wavelength:

> ********************************************************************************************
> How to query for Solar Orbiter data using the ``Wavelength`` attribute using ``sunpy-soar``.
> ********************************************************************************************

Reviewer: ```suggestion
***************************************************************
Query for Solar Orbiter data using the ``Wavelength`` attribute
***************************************************************
```

> ``sunpy-soar`` provides a convenient methods to construct queries using ``a.Wavelength`` for several different remote sensing instruments available through the SOAR.

Reviewer 2: ```suggestion
``sunpy-soar`` provides convenient methods to construct queries using ``a.Wavelength`` for several different remote sensing instruments available through the SOAR.
```

.. _sunpy-soar-how-to-query-wavelength:

*************************************
How to query using Wavelength in SOAR
*************************************

``sunpy-soar`` provides a convenient methods to query Wavelength for all different instruments available in SOAR.
In this guide, we will demonstrate how we can query for Wavelength for different instruments.
For instruments EUI, METIS and SOLOHI we get query results in form of wavelength, for instrument PHI we get query results in form of wavemin and wavemax and for instruments SPICE and STIX we do not get any meaningful data for wavelength.

For instruments EUI, METIS and SOLOHI passing a single Wavelength
=================================================================

When a single wavelength is provided it is interpreted as the wavelength.

.. code-block:: python

> 
> 

Reviewer: ```suggestion

```

         METIS metis-uv-image    L2 2023-02-01 01:30:48.680 2023-02-01 01:41:45.540     0.85      none      UVD      121.6
         METIS metis-uv-image    L2 2023-02-01 02:00:48.671 2023-02-01 02:11:44.213    12.64      none      UVD      121.6
         METIS metis-uv-image    L2 2023-02-01 02:30:48.661 2023-02-01 02:41:42.887    12.64      none      UVD      121.6
         METIS metis-uv-image    L2 2023-02-01 03:00:48.652 2023-02-01 03:11:41.560    12.64      none      UVD      121.6
         METIS metis-uv-image    L2 2023-02-01 03:30:48.643 2023-02-01 03:41:40.233    12.64      none      UVD      121.6
         METIS metis-uv-image    L2 2023-02-01 04:00:48.633 2023-02-01 04:11:38.907    12.64      none      UVD      121.6
         METIS metis-uv-image    L2 2023-02-01 04:30:38.163 2023-02-01 04:40:37.625    12.64      none      UVD      121.6
    <BLANKLINE>
    <BLANKLINE>

For instruments EUI, METIS and SOLOHI passing a range of Wavelength
===================================================================

When a range of wavelength is provided, it is interpreted as the wavemin and wavemax.

.. code-block:: python

> 
> 

Reviewer: ```suggestion

```
